scripts/libclient/medium_test.sh

```diff
@@ -14,7 +14,7 @@ setup() {
 # run処理を記述する関数
 # NOTE: この関数は例外的にワンライナーで書かなくて良い
 run() {
-    docker-compose $COMPOSE_FILES_OPT up medium-libc
+    docker-compose $COMPOSE_FILES_OPT run medium-libc pytest src/tests -s -v -log-cli-level=DEBUG --durations=5
```

Comment:
Is `--durations=5` for debug? Do you want to keep it forever?

Reply:
I wrote it because I thought that option was a very good one.
But I realized that I don't need this option except to speed up testing, so I delete it.
[af43f46](https://github.com/acompany-develop/QuickMPC/pull/197/commits/af43f46ef6d9b96131f0b6f6997b88ad3b8d59af)